SwiftMonkeyPaws/MonkeyPaws.swift

```diff
@@ -35,10 +31,60 @@ private let circleRadius: CGFloat = 7
     }
     ```
 */
+
 public class MonkeyPaws: NSObject, CALayerDelegate {
+
+    public struct Configuration {
+        // Customise the appearance of the paws
+        public struct Paws {
+
+            /// Define the colour of the Paws
+            ///
+            /// - randomized: random colour for each paw
+            /// - constant: same colour for the paws
+            enum Colour {
+                case randomized
+                case constant(UIColor)
+            }
+            // Colour of the paws
+            let colour: Colour
+
+            // Brightness of a particular paw
+            let brightness: CGFloat
+
+            // Maximum visible paws at one time
+            let maxShown: Int
+
+            public static let `default`: Paws = Paws(colour: .randomized,
+                                                     brightness: 0.5,
+                                                     maxShown: 15)
+        }
+
+        public struct Radius {
+
+            /// Radius of the cross draw upon canceling a touch event
+            let cross: CGFloat
+
+            /// Radius of the circle draw upon ending a touch event
+            let circle: CGFloat
+
+            public static let `default`: Radius = Radius(cross: 7,
```

Comment:
Please consider replacing with a parameterless public initializer with all parameters having default values (just like it was suggested for the `Paws` object).